collector/tasks_response.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+// TasksResponse is a representation of the Task management API.
+type TasksResponse struct {
```

Comment:
JSON response definitions should go into the tasks.go file. We don't need a separate file just for this.

Reply:
Done